back-end/src/routes/saveRoute.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { checkOwnership } from '../middlewares/authentication';
+import { createSave, deleteSave, readSave, readSavedBy, readUserSaves } from '../APIs/controllers';
+
+export const saveRouter = Router();
+
+saveRouter.post('/', checkOwnership('saves'), createSave);
+
+saveRouter.get('/', checkOwnership('saves'), readSave);
+
+saveRouter.get('/savedBy', readSavedBy);
```

Comment:
Why is this needed?

Reply:
At the moment it is not needed, I left it for future functionality. But you're right, I'll remove it to reduce confusion.